pylint_odoo/checkers/no_modules.py

```diff
@@ -605,11 +612,23 @@ def visit_call(self, node):
                     for x in itertools.chain(arg.args, arg.keywords or [])])
                 right = '_(%s).format(%s)' % (
                     arg.func.expr.as_string(), params_as_string)
-            if wrong:
+            if wrong and right:
                 self.add_message(
                     'translation-contains-variable', node=node,
                     args=(wrong, right))
 
+            # translation-positional-used: Check "string to translate"
+            # to check "%s %s..." used where the position can't be changed
+            str2translate = arg.as_string()
+            printf_args = (
+                misc.WrapperModuleChecker.
+                _get_printf_str_args_kwargs(str2translate))
```

Comment:
```suggestion
                misc._get_printf_str_args_kwargs(str2translate))
```

Reply:
Hi @Yajo 

This is full line:

```
misc.WrapperModuleChecker._get_printf_str_args_kwargs(str2translate))
```
`_get_printf_str_args_kwargs` comes from `WrapperModuleChecker` class.

Reply:
Oh sorry, the formatting tricked me. 🤦🏼‍♂️